docs/articles/new-expensify/expensify-card/Change-Expensify-Card-limit.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Change Expensify Card limit
+description: Increase or decrease the limit for an Expensify Card or for a group
+---
+<div id="new-expensify" markdown="1">
+
+You can set Expensify Card limits for each group in your organization, or you can set the limit per card.
+
+# Set a limit per card
+
+1. Hover over Settings, then click **Domains**.
+2. Click the name of the domain.
+3. Next to the card, click **Edit Limit**.
+4. Ensure the Custom Smart Limit toggle is enabled to be able to set a specific card limit. Otherwise, the card limit will be determined by the limit set for the group that the employee is in. 
+5. In the Limit Amount field, enter the desired limit. If set to $0, the card will be disabled for use until the limit is increased.
+6. Click **Save**. 
+
+# Set a limit per group
+
+1. Hover over Settings, then click **Domains**.
+2. Click the name of the domain.
+3. Click the **Groups** tab on the left.
+4. Next Expensify Card Smart Limit field for the card and enter the desired limit. 
```

Comment:
Seems like this is missing a word... perhaps `Next click` but not sure 😄

Reply:
Updated!